docs/docsite/rst/porting_guides/porting_guide_2.7.rst

```diff
@@ -54,7 +54,13 @@ Modules
 
 Major changes in popular modules are detailed here
 
-
+* The :ref:`DEFAULT_SYSLOG_FACILITY` configuration option tells Ansible modules to use a specific
+  `syslog facility <https://en.wikipedia.org/wiki/Syslog#Facility>`_ when logging information on the
+  managed machine.  This was broken for machines which used journald and had the python bindings for
+  journald installed.  The bug meant that machines without journald could be logging to a different
+  location than machines which did use journald.  Ansible-2.7 fixes this bug which means that the
+  location of remote logs on systems which use journald could change if
+  :ref:`DEFAULT_SYSLOG_FACILITY` is configured.
 
```

Comment:
Thanks for adding documentation to the porting guide! I'd suggest we include specific information about where the log messages were going when the bug was active. Here's a suggested revision based on information in the ticket and some slack conversations:

The :ref:`DEFAULT_SYSLOG_FACILITY` configuration option tells Ansible modules to use a specific `syslog facility <https://en.wikipedia.org/wiki/Syslog#Facility>`_ when logging information on all managed machines. Due to a bug in 2.5 and 2.6, this setting did not affect machines using ``journald`` with ``systemd-python`` installed. On those machines, Ansible log messages were sent to ``var/log/messages``, even if you set ``DEFAULT_SYSLOG_FACILITY``. Ansible 2.7 fixes this bug, routing all Ansible log messages according to the value set for ``DEFAULT_SYSLOG_FACILITY``. If you have ``DEFAULT_SYSLOG_FACILITY`` configured, the location of remote logs on systems which use ``journald`` may change.

Reply:
@acozine I updated the porting guide with a few small differences from your suggested revision.  What do you think?

Reply:
Excellent, thank you. Looks good.